Create ui-c8y-1018-503-106-hide-navigator-should-work-in-cockpit-app-6710.md (#2273)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-106-hide-navigator-should-work-in-cockpit-app-6710.md

---
date: ""

> title: `hideNavigator` should work in cockpit app (#6710)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Cockpit app now supports the hideNavigator configuration option
```

> title: Cockpit app now supports the hideNavigator configuration option

Reviewer: ```suggestion
title: Hide navigator option now works properly in the Cockpit application
```

> title: Hide navigator option now works properly in the Cockpit application

Reviewer: ```suggestion
title: Hide navigator option now works in the Cockpit application
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-60045
version: 1018.503.106
---

> `hideNavigator` should work in cockpit app (#6710)

Author: The Claude AI suggested to use the following body:
```suggestion
The cockpit app did not properly support the hideNavigator configuration option which allows hiding the navigator menu on the left side. This has been fixed so that setting hideNavigator to true in the app descriptor now properly hides the navigator menu in the cockpit app as expected. This change improves consistency of the hideNavigator option across different app types and makes it easier to control the visibility of the navigator menu in cockpit apps.
```

> The cockpit app did not properly support the hideNavigator configuration option which allows hiding the navigator menu on the left side. This has been fixed so that setting hideNavigator to true in the app descriptor now properly hides the navigator menu in the cockpit app as expected. This change improves consistency of the hideNavigator option across different app types and makes it easier to control the visibility of the navigator menu in cockpit apps.

Reviewer: ```suggestion
The Cockpit application did not properly support the `hideNavigator` configuration option which allows hiding the navigator menu on the left side. This issue has been fixed so that setting `hideNavigator` to true in the app descriptor now properly hides the navigator menu in the Cockpit application as expected. This change improves the consistency of the hideNavigator option across different applications.
```
